packages/runtime/container-runtime/src/pendingStateManager.ts

```diff
@@ -385,11 +386,12 @@ export class PendingStateManager implements IDisposable {
                         this.rollbackNextPendingState();
                     }
                 } catch(err) {
-                    const error = DataProcessingError.wrapIfUnrecognized(
-                        err,
-                        "checkpointRollback",
-                        undefined,
-                        "RollbackError: ");
+                    const error = wrapError(err, (message) => {
+                        return DataProcessingError.create(
+                            `RollbackError: ${message}`,
+                            "checkpointRollback",
+                            undefined) as DataProcessingError;
```

Comment:
Why `as DataProcessingError`?  if you need error to have that type I think you can do `wrapError<DataProcessingError>` instead.

Reply:
That doesn't work. It needs to return LoggingError but DataProcessingError.create returns IFluidErrorBase.